wled00/dmx_output.cpp

```diff
@@ -58,9 +58,6 @@ void handleDMXOutput()
         case 5:        // Shutter channel. Controls the brightness.
           dmx.write(DMXAddr, brightness);
           break;
-        case 6:        // Sets this channel to 255. Like 0, but more wholesome.
-          dmx.write(DMXAddr, 255);
-          break;
```

Comment:
What happens to people's existing config that were using case 6 - can we be backwards compatible?

Reply:
Already [pointed out by coderabbitai](https://github.com/wled/WLED/pull/4876#pullrequestreview-3155318375). Should I implement the solution suggested by it?

<img width="775" height="533" alt="image" src="https://github.com/user-attachments/assets/bd1bb378-b864-498e-95de-2b4ce9292ccb" />